vcpkg/users/binarycaching-troubleshooting.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Binary caching troubleshooting guide
+description: Troubleshooting your binary caching setup
+author: dan-shaw
+ms.author: danshaw2
+ms.date: 9/20/2023
+ms.prod: vcpkg
+---
+
+# Binary caching troubleshooting guide
+
+This guide is intended for vcpkg binary caching users that are running into unexpected behavior from this experience. Please review the [binary caching](./binarycaching.md) reference documentation for a basic understanding of what this feature does and how it works.
+
+## Set up vcpkg to print debugging information
+
+Debug mode outputs information such as the compiler verison and ABI hashes, which will help us to diagnose any binary caching issues.
+
+* [Classic mode](./classic-mode.md): add `--debug` to your command invocation.
+* CMake toolchain: set `-DVCPKG_INSTALL_OPTIONS="--debug"` in your `cmake` command call or in your `CMakePresets.json` file.
```

Comment:
I think this is OK but I also think we need an outstanding item to fix the product such that the user can do most of this without needing to turn our development debug spew.

In particular, the development debug spew is not localized.

Reply:
Agreed, that will be helpful